### ISSUE TITLE
Update netcore source image for linux/arm64

### ISSUE THREAD

**Issue by briandealwis** [GoogleContainerTools/container-debug-support#61]
In fixing #60:

The .NET debug support image is created using `mcr.microsoft.com/dotnet/core/aspnet:2.1` as the run image.  This image does not support linux/arm64.

https://github.com/GoogleContainerTools/container-debug-support/blob/f005fa30d34bb7267101d971b845558dfa250183/netcore/Dockerfile#L1-L4

I'm not sure if this run image really matters.

**Comment by briandealwis** [comment]
Using `dotnet/runtime:5.0`:
```
#10 6.698 Info: Previous installation at '/vsdbg' not found
#10 6.698 Info: Using vsdbg version '16.9.20122.2'
#10 6.699 Info: Creating install directory
#10 6.703 Using arguments
#10 6.703     Version                    : 'latest'
#10 6.703     Location                   : '/vsdbg'
#10 6.703     SkipDownloads              : 'false'
#10 6.703     LaunchVsDbgAfter           : 'false'
#10 6.703     RemoveExistingOnUpgrade    : 'false'
#10 6.710 Info: Using Runtime ID 'linux-arm64'
#10 7.106 Downloading https://vsdebugger.azureedge.net/vsdbg-16-9-20122-2/vsdbg-linux-arm64.tar.gz
#10 53.11 Info: Successfully installed vsdbg at '/vsdbg'
```

Using `dotnet/runtime:6.0` (although a preview, it's in use and aims for better support of ARM64):
```
 14.89 Info: Previous installation at '/vsdbg' not found
#10 14.89 Info: Using vsdbg version '16.9.20122.2'
#10 14.89 Info: Creating install directory
#10 14.89 Using arguments
#10 14.89     Version                    : 'latest'
#10 14.89     Location                   : '/vsdbg'
#10 14.89     SkipDownloads              : 'false'
#10 14.89     LaunchVsDbgAfter           : 'false'
#10 14.89     RemoveExistingOnUpgrade    : 'false'
#10 14.90 Info: Using Runtime ID 'linux-arm64'
#10 15.14 HTTP/2 200 
#10 15.15 Downloading https://vsdebugger.azureedge.net/vsdbg-16-9-20122-2/vsdbg-linux-arm64.tar.gz
```

So it looks like it doesn't really matter: the same version of vsdbg is installed.

**Comment by briandealwis** [comment]
Fixed by #64
